src/figuresClasses.ts

```diff
@@ -1,11 +1,102 @@
-export interface Figure {}
+// import { sign } from 'crypto';
 
-export class Triangle implements Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
```

Comment:
```suggestion
type Shape = 'triangle' | 'circle' | 'rectangle';
type Color = 'red' | 'green' | 'blue';

export interface Figure {
  shape: Shape;
  color: Color;
```
specify types more precisely